src/Orc.NuGetExplorer.Xaml/ViewModels/ExplorerPageViewModel.cs

```diff
@@ -89,7 +96,17 @@ internal class ExplorerPageViewModel : ViewModelBase, IManagerPage
             _nuGetCacheManager = nuGetCacheManager;
             _nuGetConfigurationService = nuGetConfigurationService;
             _dispatcherProviderService = dispatcherProviderService;
+
+            if (settingsProvider.Model is null)
+            {
+                throw Log.ErrorAndCreateException<InvalidOperationException>("Settings must be initialized");
+            }
+
             Settings = settingsProvider.Model;
+            if (_settings is null)
```

Comment:
Seems very duplicate (settingsProvider.Model is already checked). Maybe refactor to:

```
var settings = settingsProvider.Model;
if (settings is null)
{
    throw exception
}

Settings = settings;
```